### DISCUSSION TITLE
Feature Requests

### DISCUSSION THREAD

**Discussion by robertlong** [matrix-org/thirdroom#134] [Ideas]
Have an idea for a new Third Room feature? Add it below. If you see someone else's idea which you like, give it an upvote. This list helps us figure out what to build next.

**Comment by amytimed** [comment]
Changing the favicon of Third Room to be just the second O so it's more visible when so small

**Comment by amytimed** [comment]
Reconsider basic XR support in the MVP, as it is one of the most important things in the metaverse

**Comment by jreoka** [comment]
Screensharing

Screensharing with sound

Ability to adjust the volume!

**Comment by Akselmo** [comment]
**VRM support**

A lot of people already have VRM avatars since VRChat uses it, so being able to just upload your VRM avatar to Thirdroom would make a lot of sense in my opinion.

https://vrm.dev/en/

**Godot tools**

Godot is a game engine that is highly modifiable, and free and open source. Perhaps the Godot editor could be forked and tweaked just for making content for Thirdworld? Or alternatively, Godot plugin (this probably makes more sense :P )

https://godotengine.org/

**Comment by Thatoo** [comment]
Being able to import Mozilla Hubs and VRM avatar skeletons.

**Comment by Thatoo** [comment]
Being able to import 3D social scenes created with Spoke would be nice too to be compatible with Mozilla Hubs.

**Comment by avrame** [comment]
It would be nice to see the number of users in a space when looking at a portal.

**Comment by ja2ui0** [comment]
Invert mouse option is essential.
